common/config.js

```diff
@@ -17,11 +19,12 @@ const HOST_THE_ANVIL_PROD = 'gen3.theanvil.io';
 const HOST_THE_ANVIL_STAGING = 'staging.theanvil.io';
 const HOST_TERRA_DATA_REPO_PROD = 'data.terra.bio';
 const HOST_TERRA_DATA_REPO_STAGING = 'staging.data.terra.bio';
```

Comment:
The TDR staging hostname `staging.data.terra.bio` is incorrect. The correct TDR staging hostname is `data.staging.envs-terra.bio`

Please change from this:
```
const HOST_TERRA_DATA_REPO_STAGING = 'staging.data.terra.bio';
```
to this:
```
const HOST_TERRA_DATA_REPO_STAGING = 'data.staging.envs-terra.bio';
```